get_models_and_test.py

```diff
@@ -286,5 +283,4 @@ def mean_of_values_at_indices(arr, indices):
                             convert_file.write(json.dumps(average_resources_used, indent=4))
                             convert_file.write("\n\n=====================================\n")
                             convert_file.write("*************************************")
-                            convert_file.write("\n=====================================\n\n")
-
+                            convert_file.write("\n=====================================\n\n")
```

Comment:
If you look at the "Files changed" tab in this PR, you'll see Github is complaining about there not being a new-line at the the end of this file. Can you fix it?

Reply:
will do that.